proposals/nnnn-multiple-modules.md

```diff
@@ -493,21 +493,21 @@ A simple Module Id based permission control is listed here.
 
 - SDL shall enforce permission control. It may or may not use the local policy table depending on the implementation. The permissions are given by the user/driver via HMI, and information is passed to sdl core.
 
-- When a user launches a RC app from a phone, the app calls GetSystemCapabilites to get vehicle configurations, which includes how many rows, columns and levels are available, how many / what type / what id of remote control modules are available and where are those modules located or designed for. With this info, the app may show a map to the user so that user knows where are the control modules of each type, where the user sits. The app may ask the user to choose user's seat location (at least driver or passenger for dismissal of the lock screen) for later use. The app may get this information from other means, for example scan a RFID tag, a NFC chip or a QrCode. The user can choose a RC module or modules for remote control. 
+- When a user launches a RC app from a phone, the app calls GetSystemCapabilites to get vehicle configurations, which includes how many rows, columns and levels are available, how many / what type / what id of remote control modules are available, and where  those modules are located and/or designed for. With this info, the app may show a map to the user so that user knows where the control modules of each type are, and where the user sits. The app may ask the user to choose user's seat location (at least driver or passenger for dismissal of the lock screen) for later use. The app may get this information from other means, for example scan a RFID tag, a NFC chip or a QrCode. The user can choose a RC module or modules for remote control. 
```

Comment:
A few updates are still needed:

- Remove extra space in "...and where  those modules are located"
- Change "The app may ask the user to choose user's seat location" to "The app may ask the user to choose their seat location"

Reply:
updated